README.md

```diff
@@ -1,19 +1,15 @@
 # contour-operator
-Deploy and manage Contour using an [operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/). 
+Welcome to the Contour Operator project. Contour Operator deploys and manages Contour using an
+[operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/). 
 
-## Building
-
-To build the operator, run:
-
-```
-make manager
-```
-
-## Developing
+## Get Started
 
 ### Prerequisites
```

Comment:
Does this also need to list: `kustomize` & `kubebuilder`?

Reply:
@stevesloka good catch. Kustomize is required but I don't believe kubebuilder is. I've updated this section with the kustomize dep.